packages/analytics-js-integrations/src/integrations/GA4/browser.js

```diff
@@ -128,14 +129,11 @@ export default class GA4 {
       this.sessionNumber = sessionNumber;
     });
 
-    ScriptLoader(
-      'google-analytics 4',
-      `https://www.googletagmanager.com/gtag/js?id=${measurementId}`,
-    );
+    ScriptLoader('google-analytics 4', `${sdkBaseUrl}/gtag/js?id=${measurementId}`);
```

Comment:
Have we double-checked if adblockers are not targeting the `/gtag` path?

Reply:
I have checked with different ad-blockers, script is loading. 
I will check with some more ad-blockers.